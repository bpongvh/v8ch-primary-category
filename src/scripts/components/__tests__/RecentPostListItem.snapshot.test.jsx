import React from 'react';
import renderer from 'react-test-renderer';
import RecentPostListItem from '../RecentPostListItem';

describe('RecentPostListItem Snapshot', () => {
  it('renders correctly', () => {
    const post = {
      link: 'http://some.url.local',
      publishedAt: new Date('2018-07-08T12:00:00.000Z').toISOString(),
      title: 'Post Title',
    };
    const tree = renderer
      .create(<RecentPostListItem post={post} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
