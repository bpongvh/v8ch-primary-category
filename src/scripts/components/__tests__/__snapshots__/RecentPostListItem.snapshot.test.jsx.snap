// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecentPostListItem Snapshot renders correctly 1`] = `
<li>
  <span
    className="widget_wp-block-v8ch-primary-category__post-title"
  >
    <a
      href="http://some.url.local"
    >
      Post Title 
    </a>
  </span>
  <span
    className="widget_wp-block-v8ch-primary-category__post-date"
  >
    on  2018-7-8 19:00:00
  </span>
</li>
`;
